public/components/navigation/navigation.stache

```diff
@@ -3,22 +3,22 @@
 		<a href="/" class="pull-left main-logo">BitBalls</a>
 		<ul class="main-menu nav navbar-nav">
 		  <li role="presentation" {{#routeCurrent page='tournaments'}}class='active'{{/routeCurrent}}>
-				<a href="{{routeUrl page='tournaments'}}">Tournaments</a>
+				<a href="{{routeUrl (page='tournaments')}}">Tournaments</a>
```

Comment:
It was suggested by Matthew in his presentation to write helpers as call expressions in 3.0 wherever we could.